frontend/public/components/monitoring/receiver-forms/webhook-receiver-form.tsx

```diff
@@ -0,0 +1,40 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+export const Form = ({ formValues, handleChange }) => (
+  <div data-test-id="webhook-receiver-form" className="form-group">
+    <label className="control-label co-required">URL</label>
+    <input
+      className="pf-c-form-control"
+      type="text"
+      aria-describedby="webhook-url-help"
+      name="webhookUrl"
+      data-test-id="webhook-url"
+      value={formValues.webhookUrl}
+      onChange={handleChange}
+    />
+    <div className="help-block" id="webhook-url-help">
+      The endpoint to send HTTP POST requests to
+    </div>
+  </div>
+);
+
+export const getInitialValues = (globals, receiverConfig) => {
+  return _.isEmpty(receiverConfig)
```

Comment:
You could use `_.get`'s default value parameter like `return {webhookUrl: _.get(receiverConfig, 'url', '')}`

Reply:
your right!  I did notice that at one point but missed fixing it -thanks